tests/storages_tests/test_storages.py

```diff
@@ -93,9 +94,20 @@
     'common',  # We use a sqlite DB file for the whole experiments.
 ]
 
+
+def redis_with_flushdb():
+    # type: () -> BaseStorage
+
+    storage = RedisStorage("redis://localhost:6379/15")
+    storage.redis.flushdb()
+    return storage
+
+
 # TODO(Yanase): Replace @parametrize_storage with StorageSupplier.
 parametrize_storage = pytest.mark.parametrize(
-    'storage_init_func', [InMemoryStorage, lambda: RDBStorage('sqlite:///:memory:')])
+    'storage_init_func', [InMemoryStorage,
+                          lambda: RDBStorage('sqlite:///:memory:'),
+                          lambda: redis_with_flushdb()])
```

Comment:
I think you don't need to use a lambda function.

```suggestion
                          redis_with_flushdb])
```

Reply:
You are right. Thanks